routing/routing_integration_tests/routing_integration_tests.pro

```diff
@@ -13,6 +13,11 @@ TEMPLATE = app
 ROOT_DIR = ../..
 DEPENDENCIES = map routing search storage indexer platform geometry coding base osrm jansson protobuf tomcrypt succinct stats_client
 
+# this dependency is not built on Linux
```

Comment:
But your comment doesn't explain why this dependency is not enabled on Linux. It just claims that opening_hours is not built on Linux.